packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -299,12 +299,14 @@ angular
       };
 
       $scope.goToAssociateRepository = (domain) => {
-        const promise = $state.go('app.hosting.dashboard.git', {
-          alertId: $scope.alerts.main,
-          status: domain.status,
-          serviceName: $scope.hosting.serviceName,
-          path: domain.path,
-        });
+        const promise = $state.go(
+          'app.hosting.dashboard.multisite.git-association',
+          {
+            status: domain.status,
+            serviceName: $scope.hosting.serviceName,
+            path: domain.path,
+          },
+        );
```

Comment:
**niptick** : `$state.go(...)` should be defined in the routing file

Reply:
Another clean way (when you don't have any component) is simply to inject your function in the controller 

```
constructor( ..., goToAssociateRepository) {
     ....
     this.goToAssociateRepository = goToAssociateRepository;
}
```